daemon/api_systems.go

```diff
@@ -99,6 +105,41 @@ func getSystems(c *Command, r *http.Request, user *auth.UserState) Response {
 	return SyncResponse(&rsp)
 }
 
+type oneSystemResponse struct {
+	Model   clientutil.ModelAssertJSON `json:"model,omitempty"`
```

Comment:
I wonder if it would be more consistent to make this look like an extension on client.System but replacing the part:

Model SystemModelData `json:"model,omitempty"`

with the full headers, as I said I don't think we need to accommodate the body as we don't expect models to have bodies so far.